docs/getting-started/8-tracing/1_tracing_quickstart.ipynb

```diff
@@ -0,0 +1,4713 @@
+    "        trace_df['epoch_seconds'] = file_epoch_seconds\n",
+    "        all_trace_dfs.append(trace_df)\n",
+    "\n",
+    "    all_trace_df = pd.concat(all_trace_dfs, axis=0)\n",
+    "    return all_trace_df\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def clean_trace_dataframe(input_df):\n",
+    "    \"\"\"Clean the trace dataframe by removing all but the top-level interaction and spans\"\"\"\n",
+    "    \n",
+    "    df = input_df.copy()\n",
+    "\n",
+    "    # Add boolean indicators for rails and the top-level span. We only want to keep these\n",
+    "    df['is_rail'] = df['name'].str.startswith(\"rail\")\n",
+    "    df['is_top_span'] = df['parent_id'].isna()\n",
+    "    row_mask = df['is_rail'] | df['is_top_span']\n",
+    "    df = df[row_mask].copy()\n",
+    "\n",
+    "    # Plotly Gantt charts require a proper datatime rather than relative seconds\n",
+    "    # So use the creation-time of each trace file as the absolute start-point of the trace\n",
+    "    df['start_dt'] = pd.to_datetime(df['start_time'] + df['epoch_seconds'], unit='s')\n",
+    "    df['end_dt'] = pd.to_datetime(df['end_time'] + df['epoch_seconds'], unit='s')\n",
+    "\n",
+    "    # Print out some summary stats on how many spans and rails were found\n",
+    "    n_top_spans = df['is_top_span'].sum()\n",
+    "    n_rail_spans = df['is_rail'].sum()\n",
+    "    print(f\"Found {n_top_spans} top-level spans, {n_rail_spans} rail spans\")\n",
+    "    return df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Loading Trace Files\n",
+    "\n",
+    "Now let's load and clean the sequential and parallel data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Loaded 1 lines from parallel_trace.jsonl\n",
+      "Found 1 top-level spans, 5 rail spans\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>name</th>\n",
+       "      <th>duration</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>interaction</td>\n",
+       "      <td>8.600522</td>\n",
+       "    </tr>\n",
```

Comment:
This one looks like the overall, end-to-end latency? i.e., it's the top-tier and the rest in this table are a level below this one